modules/ROOT/pages/amazon/amazon-sqs-connector-examples.adoc

```diff
@@ -5,41 +5,47 @@ endif::[]
 :imagesdir: ../../assets/images/
 
 == Example
+//  I SUGGEST OMITTING THIS SUBTITLE, because the topic set already has a title (Amazon SQS Examples - Mule 4) and because this subtitle is at the same level as the two subtopics that it contains. Also, the title "Example" is not useful, but the two subtitles "Create a Flow..." are. //
 
 This example sends a message with metadata to an Amazon SQS queue and then receives it from the queue. The example is split into the two flows:
 
-. The first flow sends a message with metadata. Then it gets the count of the messages in the queue to validate that the message was sent.
-. The second flow receives the message and logs the message body.
-
 image::amazon/amazon-sqs-message-flow.png[sqs-message-flow]
 
-This example uses variables for some field values. You can either replace the variables with their values in the code, or you can provide the values for each variable in the `src/main/resources/mule-artifact.properties` file.
+. The first flow sends a message with metadata and then gets the count of the messages in the queue to validate that the message was sent.
+//  LIST ITEMS SHOULD BE ONE SENTENCE MAX. Additional statements, if necessary, belong in a paragraph beneath the item.  GLOBAL.  //
+. The second flow receives the message and logs the message body.
+
+This example uses variables for some field values. You can either replace the variables with actual values in the code, or you can provide the actual values for each variable in the `src/main/resources/mule-artifact.properties` file.
 
-Before you try the example, access Anypoint Studio, and check whether the Mule Palette displays entries for Amazon SNS and Amazon SQS. If not, follow the instructions in xref:amazon/amazon-sqs-connector-studio.adoc[Amazon SQS Studio Configuration] to install both connectors.
+Before you try the example, access Anypoint Studio and verify that Mule Palette displays entries for Amazon SNS and Amazon SQS. If not, follow the instructions in xref:amazon/amazon-sqs-connector-studio.adoc[Amazon SQS Studio Configuration] to install both connectors before trying this example.
+//  BECAUSE THIS IS A PREREQUISTE TO THE ENTIRE EXAMPLE, consider moving this text to the very beginning of this introduction, or creating an actual Prerequisites subheading.  //
 
```

Comment:
BECAUSE THIS IS A PREREQUISTE TO THE ENTIRE EXAMPLE, consider moving this text to the very beginning of this introduction, or creating an actual Prerequisites subheading.

Reply:
I like it better the way I have it so users know what example it is.